relayer/core/relay.go

```diff
@@ -218,11 +251,20 @@ func LoadConfig() (*Config, error) {
 	}
 	config.Eth.PrivateKey = strings.TrimPrefix(value, "0x")
 
+	// TODO: auto populate contract addresses
+	config.Eth.Contracts.PolkadotRelayChainBridge = "0x8cF6147918A5CBb672703F879f385036f8793a24"
+	config.Eth.Contracts.ValidatorRegistry = "0xB1185EDE04202fE62D38F5db72F71e38Ff3E8305"
+	// TODO: query from 'BLOCK_WAIT_PERIOD' on RelayBridgeLightClient contract
+	config.Eth.BeefyBlockDelay = 5
+
 	value, ok = os.LookupEnv("ARTEMIS_SUBSTRATE_KEY")
 	if !ok {
 		return nil, fmt.Errorf("environment variable not set: ARTEMIS_SUBSTRATE_KEY")
 	}
-	config.Sub.PrivateKey = value
+	config.Sub.Parachain.PrivateKey = value
```

Comment:
do proper config in this PR plz as it is related to general plumbing/structure of relayer

Reply:
Updated in https://github.com/Snowfork/polkadot-ethereum/pull/314/commits/ec27500ba0c88d0268ac6faaa85f59f2ab3dbffd